components/Notification.js

```diff
@@ -101,9 +101,9 @@ class ToastNotification extends Component {
     className: PropTypes.string,
     kind: PropTypes.oneOf(['error', 'info', 'success', 'warning']).isRequired,
     title: PropTypes.string.isRequired,
-    subtitle: PropTypes.string.isRequired,
+    subtitle: PropTypes.oneOfType([PropTypes.string, PropTypes.element]).isRequired,
```

Comment:
Would it be helpful instead for these props to just be `PropTypes.node`?

Reply:
@joshblack Yep that makes more sense. I'll swap them